validator/validator-main.protoascii

```diff
@@ -867,6 +867,25 @@ tags: {
     dispatch_key: NAME_VALUE_DISPATCH
   }
 }
+# AMP metadata, name=amp-cta-landing-page-type
+# Specifies the Single Page Story Ad landing page type
+tags: {
+  html_format: AMP4ADS
+  tag_name: "META"
+  spec_name: "meta name=amp-cta-landing-page-type"
+  mandatory_parent: "HEAD"
+  unique: true
+  attrs: {
+    name: "content"
+    mandatory_oneof: ['AMP', 'NOAMP', 'STORY']
+  }
+  attrs: {
+    name: "name"
+    mandatory: true
+    value_casei: "amp-cta-landing-page-type"
+    dispatch_key: NAME_VALUE_DISPATCH
```

Comment:
I don't understand the comments in validator.proto well enough to know if this is needed. Can you add a positive test and a negative test for this change?

Reply:
this is necessary. but tests are good as well

Reply:
@honeybadgerdontcare just for my knowledge could you clarify what this is actually doing?

Reply:
I guess the comment isn't clear enough in [validator.proto](https://github.com/ampproject/amphtml/blob/master/validator/validator.proto#L136).

In essence it is telling the validator that this tag spec for this tag, when it has this attribute with this attribute name and attribute value pair should be evaluated first before any other tagspec with this tag and this attribute name. Only one tagspec for this tag with this attribute name and attribute value may have `dispatch_key: NAME_VALUE_DISPATCH`. It's giving a hint to the validator that it's the more relevant tagspec.

Does that help?

Reply:
Yes, thanks. Added some tests.